src/AppInstallerSharedLib/Public/AppInstallerLogging.h

```diff
@@ -48,20 +48,22 @@ namespace AppInstaller::Logging
     // Channels enable large groups of logs to be enabled or disabled together.
     enum class Channel : uint32_t
     {
-        Fail,
-        CLI,
-        SQL,
-        Repo,
-        YAML,
-        Core,
-        Test,
-        Config,
+        Fail = 0x1,
+        CLI = 0x2,
+        SQL = 0x4,
+        Repo = 0x8,
+        YAML = 0x10,
+        Core = 0x20,
+        Test = 0x40,
+        Config = 0x80,
         // Put special channel semantics at the end to preserve the bitspace.

```

Comment:
I don't really get this comment. Was it about the computed masks with bit shifts?

Reply:
```suggestion
```